website/content/en/docs/contribution-guidelines/releasing.md

```diff
@@ -4,391 +4,275 @@ linkTitle: Releasing
+Also create and merge a commit that updates the top-level [Makefile] variable `IMAGE_VERSION`
+to the upcoming release tag `v1.3.0`. This variable ensures sample projects have been tagged
+correctly prior to the release commit.
 
-Make sure you've [uploaded your GPG key][link-github-gpg-key-upload] and configured git to [use that signing key][link-git-config-gpg-key] either globally or for the Operator SDK repository. Tagging will be handled by `release.sh`.
+### 1. Create and push a release commit
 
-**Note:** the email the key is issued for must be the email you use for git.
+Create a new branch to push the release commit:
 
 ```sh
-$ git config [--global] user.signingkey "$GPG_KEY_ID"
-$ git config [--global] user.email "$GPG_EMAIL"
+git checkout master
+git pull
+git checkout -b release-v1.3.0
 ```
 
-Also, make sure that you setup the git gpg config as follows.
-```console
-$ cat ~/.gnupg/gpg.conf
-default-key $GPG_KEY_ID
-```
+Run the pre-release `make` target:
 
-**NOTE** If you do a release from an OSX machine, you need to configure `gnu-gpg` to sign the release's tag:
-- Install the requirements by running: `brew install gpg2 gnupg pinentry-mac`
-- Append the following to your ~/.bash_profile or ~/.bashrc or ~/.zshrc
-```sh
-export GPG_TTY=`tty`
-```
-- Restart your Terminal or source your ~/.\*rc file
-- Then, make sure git uses gpg2 and not gpg
-```sh
-$ git config --global gpg.program gpg2
-```
-- To make sure gpg2 itself is working
 ```sh
-$ echo "test" | gpg2 --clearsign
+make prerelease RELEASE_VERSION=v1.3.0
 ```
 
-## Release branches
-
-Each minor release has a corresponding release branch of the form `vX.Y.x`, where `X` and `Y` are the major and minor
-release version numbers and the `x` is literal. This branch accepts bug fixes according to our [backport policy][backports].
+The following changes should be present:
 
-After the minor release is made, this branch must be fast-forwarded to that release's tag and a post-release PR made
-against this branch. See the [release process](#8-create-a-pr-for-post-release-version-updates) for more details.
-
-#### Cherry-picking
+- `changelog/generated/v1.3.0.md`: commit changes (created by changelog generation).
+- `changelog/fragments/*`: commit deleted fragment files (deleted by changelog generation).
+- `website/content/en/docs/upgrading-sdk-version/v1.3.0.md`: commit changes (created by changelog generation).
+- `website/config.toml`: commit changes (modified by release script).
 
-Once a minor release is complete, bug fixes can be merged into the release branch for the next patch release.
-Fixes can be added automatically by posting a `/cherry-pick v1.3.x` comment in the `master` PR, or manually by running:
+Commit these changes and push:
 
 ```sh
-$ git checkout v1.3.x
-$ git checkout -b cherrypick/some-bug
-$ git cherry-pick "$GIT_COMMIT_HASH" # Hash of the merge commit to master.
-$ git push upstream cherrypick/some-bug
+git add --all
+git commit -m "Release v1.3.0"
```

Comment:
Would it be bad for the `make prerelease` target to do the git add?

```make
prerelease:
    # do the stuff
    git add changelog website
    # show all the files added 
    git status
    # OR show the diff
    git diff --cached
    echo "If things look fine run: git commit -m \"Release ${RELEASE_VERSION}\"\ngit push -u origin release-${RELEASE_VERISON}"
```

Reply:
I'll add this in a follow-up.